.clang-format

```diff
@@ -82,10 +76,10 @@ ObjCBlockIndentWidth: 4
 ObjCSpaceAfterProperty: true
 ObjCSpaceBeforeProtocolList: true
 PenaltyBreakBeforeFirstCallParameter: 19
-PenaltyBreakComment: 300
-PenaltyBreakFirstLessLess: 120
-PenaltyBreakString: 1000
-PenaltyExcessCharacter: 1000000
+PenaltyBreakComment: 80
+PenaltyBreakFirstLessLess: 80
+PenaltyBreakString: 80
+PenaltyExcessCharacter: 80
```

Comment:
Penalty is not really a limit, it's a penalty and we need to find a balance.

Reply:
ok